addons/pharma/stringtable.xml

```diff
@@ -3296,6 +3313,7 @@
             <Dutch>Blijf verbinden tot uit het systeem</Dutch>
             <French>Continuer à bander jusqu'à ce que le produit soit éliminé du système</French>
             <Italian>Continua a fasciarti finché non esce dal sistema</Italian>
+            <German>Bleiben Sie so lange verbunden, bis das System außer Betrieb ist</German>
```

Comment:
This is incorrect grammar, "So lange verbinden, bis es aus dem Kreislauf ist" could be better, still feels rough though